src/components/20-molecules/list/child.scss

```diff
@@ -0,0 +1,40 @@
+axa-list {
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+
+    & > li {
+        margin: 8px 0;
+        padding-left: 36px;
+        // This SVG renders a bullet point for unordered lists.
+        background-image: url('data:image/svg+xml;charset=UTF-8,<svg xmlns="http://www.w3.org/2000/svg" width="4" height="4" viewBox="0 0 4 4" fill="none"><rect width="4" height="4" rx="2" fill="black"/></svg>');
```

Comment:
Dont we want the browser default?

Reply:
I mean, the bullet points. We dont need a svg for that. <ul> is renderd with bullets by default?!